Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WMRDevice.cs

```diff
@@ -50,10 +60,25 @@ private void InitializeSources()
             }
         }
 
+        //TODO - kept for reference - clean later.
+        //private void SpatialManager_SourcePressed(Windows.UI.Input.Spatial.SpatialInteractionManager sender, Windows.UI.Input.Spatial.SpatialInteractionSourceEventArgs args)
```

Comment:
Future note: We may need to confirm the API contract / OS version to ensure that functionality is present. I can work with you on that if needed.

Reply:
Good thing we already have a contract checker class.  @davidkline-ms do you know if that contract checking uses reflection or something like that under the hood?